data/json/items/tools.json

```diff
@@ -966,21 +966,6 @@
       "skill2": "computer"
     }
   },
-  {
-    "id": "brazier",
-    "type": "TOOL",
-    "name": "brazier",
-    "description": "This is a large metal stand with slots on the side.  You can set it somewhere through the construction menu and then set fires in it with no risk of spreading.",
```

Comment:
The original description is much better, it's more detailed and contains a hint about the specific properties of a brazier (prevents fire from spreading).